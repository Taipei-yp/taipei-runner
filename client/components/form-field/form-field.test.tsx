import React from "react";
import TestRenderer, { act } from "react-test-renderer";
import { FormField } from ".";

describe("Component FormField", () => {
  const formField = TestRenderer.create(
    <FormField labelText="test">
      <input type="text" />
    </FormField>,
  );

  it("correct render", () => {
    expect(formField.toJSON()).toMatchSnapshot();
  });
});
