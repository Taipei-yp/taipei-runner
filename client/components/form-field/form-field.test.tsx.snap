// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component FormField correct render 1`] = `
<div
  className={
    Object {
      "has": [Function],
      "is": [Function],
      "mix": [Function],
      "split": [Function],
      "state": [Function],
      "toString": [Function],
    }
  }
>
  <label
    className={
      Object {
        "has": [Function],
        "is": [Function],
        "mix": [Function],
        "split": [Function],
        "state": [Function],
        "toString": [Function],
      }
    }
  >
    test
  </label>
  <div
    className={
      Object {
        "has": [Function],
        "is": [Function],
        "mix": [Function],
        "split": [Function],
        "state": [Function],
        "toString": [Function],
      }
    }
  >
    <input
      type="text"
    />
  </div>
</div>
`;
